pkg/controller/logcollector/logcollector_controller.go

```diff
@@ -170,18 +170,47 @@ func GetLogCollector(ctx context.Context, cli client.Client) (*operatorv1.LogCol
 	return instance, nil
 }
 
+func fillDefaults(instance *operatorv1.LogCollector) []string {
```

Comment:
Please add a comment on this function that includes what the slice returned contains.